docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -214,6 +214,30 @@ offsets as reported by Kafka, the consumer lag per partition, as well as the agg
 consumer lag per partition may be reported as negative values if the supervisor has not received a recent latest offset
 response from Kafka. The aggregate lag value will always be >= 0.
 
+The status report also contains the supervisor's state and a list of recently thrown exceptions (whose max size can be 
+controlled using the `druid.supervisor.stream.maxStoredExceptionEvents` config parameter).  The list of states is as
+follows:
+
+|State|Description|Priority|
+|-----|-----------|--------|
+|UNHEALTHY_SUPERVISOR|The supervisor has encountered non-transient errors on the past `druid.supervisor.stream.unhealthinessThreshold` iterations|1|
+|UNHEALTHY_TASKS|The last `druid.supervisor.stream.taskUnhealthinessThreshold` tasks have all failed|2|
+|UNABLE_TO_CONNECT_TO_STREAM|The supervisor is encountering connectivity issues with Kafka and has not successfully connected in the past|3|
+|LOST_CONTACT_WITH_STREAM|The supervisor is encountering transient connectivity issues with Kafka but has successfully connected in the past|4|
```

Comment:
nit: For `LOST_CONTACT_WITH_STREAM`, are the connectivity issues necessarily transient? Maybe sometimes the connectivity never comes back without operator intervention.